git-bz

```diff
@@ -777,8 +786,14 @@ class BugAttachment:
             # If we have a raw diff, add the From/Date/Subject lines to help git
             # detect the patch format.
             if diff.startswith('diff '):
-                self.data = "From: %s\nDate: %s\nSubject: %s\n%s" % (
-                    raw_attachment['creator'],
+                real_name = creator['real_name']
+                # Normally brackets, e.g., [], (), are used along with the real
+                # user name to represent the user's status/timezone. Remove any
+                # of them before applying to the patch.
+                real_name = re.split(r"[\[|\(]", real_name)[0].strip()
```

Comment:
You don't need the | in the regular expression pattern, unless you are also trying to match against |.

You could also add ‹ and : to this, which would handle glob and dbaron's Bugzilla name.